.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: enhancement
+assignees: ''
+
+---
+
+**Relevant parts**
+
+Backend:
+- [ ] Kaldi
+- [ ] Whisper
+- - [ ] Ctranslate
+- - [ ] Torch
+- [ ] Nemo
+
+Serving mode:
+- [ ] Task
+- [ ] Websocket
+- [ ] HTTP
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
"I'm always frustrated when" does not sound to me like the user is going to explain clearly and concisely his problems.
I would remove this example.